integration-tests/resources/wallets/transfer_from_pos_vault.rho

```diff
@@ -0,0 +1,24 @@
+new return, rl(`rho:registry:lookup`), poSCh, stdout(`rho:io:stdout`), 
+     deployerId(`rho:rchain:deployerId`), transferRetCh, log in {
+   rl!(`rho:rchain:pos`, *poSCh) |
+   match (
+    "%TARGET_ADDR",
+    %AMOUNT
+   ) {
+       (targetAddr, amount) => {
+         for(@(_, PoS) <- poSCh) {
+           @PoS!("humanTransfer", targetAddr, amount, *deployerId, *transferRetCh) |
```

Comment:
Sorry. I made one more mistake on the issue. It is not `humanTransfer` which is obsolete.
You should use `posVaultTransfer`.